src/vs/workbench/services/policies/common/accountPolicyService.ts

```diff
@@ -0,0 +1,49 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { IStringDictionary } from '../../../../base/common/collections.js';
+import { ILogService } from '../../../../platform/log/common/log.js';
+import { AbstractPolicyService, IPolicyService, PolicyDefinition } from '../../../../platform/policy/common/policy.js';
+import { DefaultAccountService, IDefaultAccountService } from '../../accounts/common/defaultAccount.js';
+
+export class AccountPolicyService extends AbstractPolicyService implements IPolicyService {
+	private chatPreviewFeaturesEnabled: boolean = true;
+	constructor(
+		@ILogService private readonly logService: ILogService,
+		@IDefaultAccountService private readonly defaultAccountService: DefaultAccountService
+	) {
+		super();
+		this._register(this.defaultAccountService.onDidChangeDefaultAccount((account) => {
```

Comment:
Are we missing initialisation here? What if the default account is already available? Then, there wont be a change event triggered for default account change.

Reply:
added usage of `this.defaultAccountService.getDefaultAccount()`